### DISCUSSION TITLE
inline method logs location

### DISCUSSION THREAD

**Discussion by mohan-nagandlla** [strimzi/strimzi-kafka-operator#5746] [General]
HI team I just use the inline method to capture the logs for kafka connect but where are these logs are storing, May I know the path please.

**Comment by scholzj** [comment]
Isn't this duplicate of #5744?

**Reply by mohan-nagandlla** [reply]
yes and i just want to know like shall we able to persist the logs for kafka connect

**Reply by scholzj** [reply]
Well, I replied on the issue which you opened. Logging is done to stdout where your container engine should store it and you can collect it with tools such as Fluentd etc.
